app/src/main/java/ch/admin/bag/dp3t/networking/ConfigWorker.kt

```diff
@@ -95,6 +98,15 @@ class ConfigWorker(context: Context, workerParams: WorkerParameters) : Coroutine
 				}
 			} else {
 				cancelNotification(context)
+
+				if (config.isCheckInUpdateNotificationEnabled && !secureStorage.checkInUpdateNotificationShown
+					&& secureStorage.onboardingCompleted && secureStorage.lastShownUpdateBoardingVersion < CHECKIN_UPDATE_BOARDIN_VERSION
+					&& !secureStorage.forceUpdateLiveData.hasObservers()
```

Comment:
Why is this hasObservers() check necessary?

Reply:
if you open the app (and trigger the config-request from there) we do not want to show a notification

Reply:
Does this work, if the onboarding activity is opened at app start? As far as I have seen, the forceUpdateLiveData is only observed by the MainActivity.

Reply:
when I tested it it worked, yes. And in most cases the configrequest will happen in the background anyway and not when people open the app.